cmd/syncEnumerator.go

```diff
@@ -62,7 +62,7 @@ func (cca *cookedSyncCmdArgs) initEnumerator(ctx context.Context) (enumerator *s
 		if entityType == common.EEntityType.File() {
 			atomic.AddUint64(&cca.atomicSourceFilesScanned, 1)
 		}
-	}, nil)
+	}, nil, false)
```

Comment:
>false [](start = 9, length = 5)

We should perhaps support preserving tags for sync.

Reply:
That's a big TODO. I'm working on it.

Reply:
Turned out it wasn't such a big change. Done